chain/store_test.go

```diff
@@ -23,7 +25,201 @@ import (
 	"github.com/filecoin-project/go-filecoin/types"
 )
 
-// Note: many of these tests use the test chain defined in the init function of default_syncer_test.
+// This setup stuff is a horrendous legacy holdover from when these store tests shared a
+// common monolithic setup with the syncer tests. There's no good reason for it to still exist.
```

Comment:
This is just a direct move, indicating that the syncer tests no longer use this.